src/Tribe/Integrations/WPML/Option.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Handles the translation of the plugin settings.
+ *
+ * @since   TBD
+ *
+ * @package Tribe__Events__Integrations__WPML__Option
+ */
+
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 4f78b643acdf15d4eddac11429e4d988</sub>